projects/dotcms-field-elements/src/components/dot-form/dot-form.tsx

```diff
@@ -13,10 +13,17 @@ export class DotFormComponent {
 
     @Event() onSubmit: EventEmitter;
 
+    /** (optional) List of fields (variableName) separated by comma, to be shown */
     @Prop() fieldsToShow: string[] = [];
+
+    /** (optional) Text to be rendered on Reset button */
     @Prop({ reflectToAttr: true }) resetLabel = 'Reset';
+
+    /** (optional) Text to be rendered on Submit button */
     @Prop({ reflectToAttr: true }) submitLabel = 'Submit';
-    @Prop({ mutable: true }) fields: DotCMSContentTypeField[] = [];
+
+    /** Fields metada to be rendered */
+    @Prop({ mutable: true, reflectToAttr: true }) fields: DotCMSContentTypeRow[] = [];
```

Comment:
```suggestion
    @Prop({ mutable: true, reflectToAttr: true }) layout: DotCMSContentTypeRow[] = [];
```

Why this is mutable?